test/test_transforms.py

```diff
@@ -1,6 +1,7 @@
 import torch
 import torchvision.transforms as transforms
 import unittest
+from parameterized import parameterized
```

Comment:
@alykhantejani @fmassa I've added this dependency which I think that is useful to handle different test cases

Reply:
Not sure that we want to add another dependency. In the main `pytorch` repo these cases are handled by looping over the various parameterizations. It's not ideal, but does keep the dependency count down